fabulaws-config.yml

```diff
@@ -196,3 +195,66 @@
     - build-essential
     - stunnel4
     - pgbouncer
+
+  db_settings:
```

Comment:
It looks like I had more comments here, plus adding 0.0.0.0 to pg_networks.  I've merged from develop and updated the pull so it's more obvious what changes are being made.